apps/src/templates/sectionProgress/multiGridConstants.js

```diff
@@ -14,6 +14,9 @@ export const progressStyles = {
   lessonNumberHeading: {
     margin: '9px 16px',
     fontFamily: '"Gotham 5r", sans-serif',
+    ':hover': {
```

Comment:
A comment here explaining that if you use these styles, the component will need Radium would be useful